src/bugbug/data/bugbug_data/retriever.py

```diff
@@ -50,7 +50,8 @@ def retrieve_commits(self):
 
         logger.info('mozilla-central cloned')
 
-        repository.download_commits(self.repo_dir)
+        two_years_and_six_months_ago = datetime.utcnow() - relativedelta(years=2, months=6)
```

Comment:
Is this code update wanted in the dep update ?

Reply:
It's needed after the update because the `download_commits` function changed.